roles/confluent.variables/defaults/main.yml

```diff
@@ -199,8 +195,8 @@ zookeeper_client_port: "{{'2182' if zookeeper_ssl_enabled|bool else '2181'}}"
 ### SASL Mechanism for Zookeeper Server to Server and Server to Client Authentication. Options are none, kerberos, digest. Server to server auth only working for digest-md5
 zookeeper_sasl_protocol: "{{sasl_protocol if sasl_protocol == 'kerberos' else 'none'}}"
 
-zookeeper_truststore_path: "/var/ssl/private/zookeeper.truststore.jks"
-zookeeper_keystore_path: "/var/ssl/private/zookeeper.keystore.jks"
+zookeeper_truststore_path: "/var/ssl/private/zookeeper.truststore.p12"
+zookeeper_keystore_path: "/var/ssl/private/zookeeper.keystore.p12"
```

Comment:
this change is problematic... found that when jks files are named with p12 kafka/zookeeper fails to load them in
mostly problematic with provided keystore/truststore

anyhow this change isnt necessary to get fips to work, can be saved for later work